.github/workflows/linux_riscv_xuantie_dev_cpu.yml

```diff
@@ -2,13 +2,21 @@ name: Linux RISC-V with Xuantie (Ubuntu 22.04, Python 3.10)
 on:
   workflow_dispatch:
     inputs:
-      testFilter:
-        description: 'Filter for google tests'
+      testFilterType:
+        description: 'Filter for google tests {SMOKE | CONCISE}'
         required: true
-        default: '*smoke_AdaPoolAvg4DLayoutTest*'
+        default: 'CONCISE'
   pull_request:
     paths:
       - '.github/workflows/linux_riscv_xuantie_dev_cpu.yml'
+      - 'src/plugins/intel_cpu/src/emitters/plugin/aarch64/**'
```

Comment:
It is better to configure smart CI for that, @akladiev what do you think?

Reply:
Or we make this pipeline non-required (taking a risk of breaking changes)

Reply:
The main reason of this change is checking of outside contributions. Non - required looks reasonable.